.github/workflows/full_build.yml

```diff
@@ -49,8 +57,19 @@ jobs:
           distribution: 'temurin'
           java-version: '17.0.7'
 
+      - name: Set Up Ballerina
+        uses: ballerina-platform/setup-ballerina@v1.1.0
+        with:
+          version: '2201.9.1'
```

Comment:
Dont we need to change the version into U10

Reply:
Let's do this in a separate PR. Hope we've updated it for other components.

Reply:
Will send that once this PR is merged.